doc/user_guide/Running_in_Webassembly.md

```diff
@@ -0,0 +1,176 @@
+# Running in Webassembly
+
+Panel lets you write dashboards and other applications in Python that are accessed using a web browser. Typically, the Python interpreter runs as a separate Jupyter or Bokeh server process, communicating with JavaScript code running in the client browser. However, it is now (as of 2022) possible to run Python directly in the browser, with no separate server needed!
+
+The underlying technology involved is called [WebAssembly](https://webassembly.org/) (or WASM). More specifically, [Pyodide](https://pyodide.org/) pioneered the ability to install Python libraries, manipulate the web page's [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction) from Python, and execute regular Python code entirely in the browser. A number of libraries have sprung up around Python in WASM, including [PyScript](https://pyscript.net/).
+
+Panel can be run directly in Pyodide and has special support for rendering in PyScript. This guide will take us through the process of installing Panel in the browser, using it to render individual components or export entire applications to run as pure client applications, with no server at all (just like JavaScript).
+
+## Installing Panel in the browser
+
+To install Panel in the browser you merely have to use the installation mechanism provided by each supported runtime:
+
+### Pyodide
+
+Currently the best supported mechanism for installing packages in Pyodide is `micropip`, which can be imported within the Pyodide runtime. Once imported simple use `micropip.install`:
+
+```python
+import micropip
+micropip.install('panel')
+```
+
+To get started with Pyodide simply follow their [Getting started guide](https://pyodide.org/en/stable/usage/quickstart.html). Note that if you want to render Panel output you will also have to load Bokeh.js and Panel.js from CDN. The most basic pyodide application therefore looks like this:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/pyodide/v0.21.2/full/pyodide.js"></script>
+
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.3.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.3.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.3.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@holoviz/panel@0.14.0/dist/panel.min.js"></script>
+
+  </head>
+  <body>
+    <script type="text/javascript">
+      async function main(){
```

Comment:
Make the example able to run something like outputting `hello world` in a `div`. I know from experience that users might not always know how to turn an indicative example into a working example.